lightning/src/ln/features.rs

```diff
@@ -725,6 +743,32 @@ impl<T: sealed::Context> Features<T> {
 		}
 		true
 	}
+
+	/// Sets a custom feature bit. Errors if `bit` is outside the custom range as defined by
+	/// [bLIP 2] or if it is a known `T` feature.
```

Comment:
What is a known `T` feature bit that's outside the reserved 0-255 range for BOLTs? Would it be something like feature bits documents in [blips](https://github.com/lightning/blips/blob/master/blip-0002.md#feature-bits) like `hosted_channels`?

Reply:
I believe it's just a custom feature bit that LDK explicitly knows about and would already handle.

https://github.com/lightningdevkit/rust-lightning/blob/72b51caeba904abe677862ff1dd14c32c057b713/lightning/src/ln/features.rs#L86-L92

Reply:
Which would probably be already documented as a bLIP, yeah